packager/media/base/stream_info.h

```diff
@@ -36,6 +36,8 @@ enum Codec {
   kCodecVP8,
   kCodecVP9,
   kCodecVideoMaxPlusOne,
+  kCodecMha1,
+  kCodecMhm1,
```

Comment:
Can you move them to audio codecs section below? (after line 55)

Reply:
Moved.